src/flows/context/flow.query.tsx

```diff
@@ -238,19 +243,31 @@ export const FlowQueryProvider: FC = ({children}) => {
   }
 
   const query = (text: string, override?: QueryScope): Promise<FluxResult> => {
+    // DLW FIXME TODO:
+    // this override can be the individual range on a pipe
+    // but it's not a TimeRange time. have to reach further into the code.
+    console.log('override', override)
+    console.log('flow.query > query', flow.range)
```

Comment:
reminder to myself. Have to finish updating all the referring code, which are not using the TimeRange abstraction.